src/components/antdComponents/antdform/AntdForm.tsx

```diff
@@ -0,0 +1,59 @@
+// Imports
+import { useAuth0 } from '@auth0/auth0-react';
+import { Form, Button } from 'antd';
+// Local Imports
+import AutoCompleteForm from './AutoCompleteForm';
+
+interface AntdFormProps {
+  data: {
+    name: string;
+    // eslint-disable-next-line
+    rules: any[];
+    content: JSX.Element;
+  }[];
+}
+
+const AntdForm = (props: AntdFormProps) => {
+
+  const { user } = useAuth0();
+
+  // Inside this function we call api service. Need to check if the SYMBL exists
+  const onSubmit = (values) => {
+    console.log('Success:', values, user?.sub);
+  };
+
+  const onSubmitFailed = (errorInfo) => {
+    console.log('Failed:', errorInfo, user?.sub);
+  };
+
+  const displayForm = props.data.map((form) => (
+    <Form.Item name={form.name} key={form.name} rules={form.rules}> 
+      {form.content}
+    </Form.Item>
+  ));
+  
+
+  return (
+    <div className='genericContainer'>
+      <Form 
```

Comment:
the form should be inside the inner container else it wont get the padding. unless there is a specific reason for this?

Reply:
Yep form is the outer component. All of the Form Items are inside the Inner container. I had to do it this way to get the button outside of the Inner container whilst also keeping it semi grouped.